### ISSUE TITLE
Ending before starting error

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#152]
<div><strong>This discussion was imported from <a href="http://outlookgooglecalendarsync.codeplex.com/discussions/656869" target="_blank">CodePlex</a></strong></div><hr/>
<div>
<p><strong><a href="http://www.codeplex.com/site/users/view/jdw5155" target="_blank">jdw5155</a></strong> wrote at 2016-08-02 15:49:</p><div class="markDownOutput ">Nice tool you have here. I've only encountered one recurring issue and I hope you can help. When OGCS syncs from Google to Outlook there is apparently an event (probably recurring) that is malformed on Google Calendar that is triggering the following error:
<br>
<br>
WARNING: Appointment update failed. <br>
The end date you entered occurs before the start date. <br>
<br>
I'd fix the offending events but I don't know which ones they are. How can I see which events are the problem or should I make a bug report to ask that the description of the offending event be added to the above error?
<br>
<br>
Thanks in advance! <br>
Jim<br>
</div></div>
<hr/>
<div>
<p><strong><a href="http://www.codeplex.com/site/users/view/phw198" target="_blank">phw198</a></strong> wrote at 2016-08-11 16:05:</p><div class="markDownOutput ">Hi <br>
<br>
Yes, this is an issue I'm aware of and have a fix half-coded - should be in the next alpha release, hopefully this weekend.
<br>
<br>
Basically, it's due to updating the time before the timezone, which can make the end temporarily seem like it's before the start. The workaround would be to delete the entire appointment in Outlook and let it recreate fully, or to manually change the times
 to match Google. If you check the &quot;Verbose output&quot; on the Sync tab you should be able to see what appointment is having the problem.
<br>
<br>
Cheers, <br>
Paul<br>
</div></div>
